### ISSUE TITLE
Provide variable fonts?

### ISSUE THREAD

**Issue by abcfy2** [adobe-fonts/source-han-mono#5]
### Prerequisites

* [x] If you are reporting an issue that also affects [*Source Han Sans*](https://github.com/adobe-fonts/source-han-sans/) from which *Source Han Mono* was derived, please report the issue [there](https://github.com/adobe-fonts/source-han-sans/issues) instead. If you do not, you are hereby warned that your issue will be closed with extreme prejudice.
* [x] If you are requesting that region-specific subset OTFs or other deployment formats be produced, such as weight-specific OTCs, please make them yourself. The 70 individual language-specific OTFs that were used to build the 70-font OpenType/CFF Collection are considered "source" files, and are therefore available loose in the build directories. Also, the COMMANDS.txt file provides the command lines for rebuilding the 70 source OTFs and the 70-font OTC.
* [x] Thoroughly search the [open](https://github.com/adobe-fonts/source-han-mono/issues) and [closed](https://github.com/adobe-fonts/source-han-mono/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue. If in doubt, search again.
* [x] Read the [ReadMe](https://github.com/adobe-fonts/source-han-mono/raw/master/SourceHanMonoReadMe.pdf) (PDF) to better understand the scope of the project, to include the *Known Issues* section at the very end. If you feel that your enhancement request is probably beyond the scope of this project, chances are it is.

### Description

Both Source Han Sans and Source Code Pro have variable font.

Would you please also provide variable fonts for Source Han Mono?

**Comment by punchcutter** [comment]
Yes, this will happen at some point, but there's no planned timeline right now.
